python/ray/tests/test_client.py

```diff
@@ -331,16 +338,27 @@ def __init__(self):
 
             def say_hello(self, whom):
                 self.count += 1
-                return ("Hello " + whom, self.count)
+                return "Hello " + whom, self.count
+
+            @ray.method(num_returns=2)
+            def say_hi(self, whom):
+                self.count += 1
+                return "Hi " + whom, self.count
 
         actor = HelloActor.remote()
         s, count = ray.get(actor.say_hello.remote("you"))
         assert s == "Hello you"
         assert count == 1
-        s, count = ray.get(actor.say_hello.remote("world"))
+
+        ref = actor.say_hello.remote("world")
+        s, count = ray.get(ref)
         assert s == "Hello world"
         assert count == 2
 
+        r1, r2 = actor.say_hi.remote("ray")
+        assert ray.get(r1) == "Hi ray"
+        assert ray.get(r2) == 3
```

Comment:
Is this to improve the test? Shouldn't it behave like it was before making this change?

Reply:
(just double checking to make sure the async behavior did not introduce an issue)

Reply:
Yes, this is an enhancement to ensure the behavior is the same between cluster and client modes, for `@ray.method(num_returns=2)`.